OptimoveSDK/Classes/Legacy/Logger/OptiLoggerStreamsContainer.swift

```diff
@@ -0,0 +1,75 @@
+//  Copyright © 2019 Optimove. All rights reserved.
+
+import Foundation
+import OptimoveCore
+
+/// Contains an array of a log receivers.
+@objc public final class OptiLoggerStreamsContainer: NSObject {
```

Comment:
@eligoptimove Why do we still need this one?

Reply:
This is a public API

Reply:
Got it, can you mark it deprecated to that we could rename it in V3?